tests/test_core.py

```diff
@@ -1236,15 +1236,16 @@ def test_exceptions_3(self):
     self.do_run('src.js', 'Caught exception: Hello\nDone.', args=['2'], no_build=True)
 
   def test_exceptions_allowed(self):
-    self.set_setting('DISABLE_EXCEPTION_CATCHING', 2)
     # Wasm does not add an underscore to function names. For wasm, the
     # mismatches are fixed in fixImports() function in JS glue code.
     self.set_setting('EXCEPTION_CATCHING_ALLOWED', ["_Z12somefunctionv"])
     # otherwise it is inlined and not identified
     self.set_setting('INLINING_LIMIT', 50)
 
     self.do_core_test('test_exceptions_allowed.cpp')
-    size = len(open('test_exceptions_allowed.js').read())
+    size = os.path.getsize('test_exceptions_allowed.js')
+    if self.is_wasm():
+      size += os.path.getsize('test_exceptions_allowed.wasm')
```

Comment:
This PR seems to contain the contents of #13734, here and several elsewhere in this file.

Reply:
Yes, there are stacked but both targetting main.   Now that #13734,has landed the diff should look cleaner..